extensions/notebook/src/common/utils.ts

```diff
@@ -23,20 +23,16 @@ export function getLivyUrl(serverName: string, port: string): string {
 }
 
 export async function mkDir(dirPath: string, outputChannel?: vscode.OutputChannel): Promise<void> {
-	if (!await fs.pathExists(dirPath)) {
-		if (outputChannel) {
-			outputChannel.appendLine(localize('mkdirOutputMsg', "... Creating {0}", dirPath));
-		}
-		await fs.ensureDir(dirPath);
+	if (outputChannel) {
+		outputChannel.appendLine(localize('ensureDirOutputMsg', "... Ensuring {0} exists", dirPath));
 	}
+	await fs.ensureDir(dirPath);
```

Comment:
Would suggest renaming this function to `ensureDir` then to match the call being done.